mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -171,6 +178,22 @@ func ResourceSqlDatabaseInstance() *schema.Resource {
 								},
 							},
 						},
+						"data_cache_config": {
+                            Type:        schema.TypeList,
+                            Optional:    true,
+                            MaxItems:    1,
+                            Description: `Data cache configurations.`,
+                            Elem: &schema.Resource{
+                                Schema: map[string]*schema.Schema{
+                                    "data_cache_enabled": {
+                                        Type:        schema.TypeBool,
+                                        Optional:    true,
+                                        Default:     false,
+                                        Description: `Whether data cache is enabled for the instance.`,
+                                    },
+                                },
+                            },
```

Comment:
Looks like this requires code formatting.

Reply:
Done